src/test/config.sol

```diff
@@ -1722,39 +1724,9 @@ contract Config {
             calc_cut:     9900,
             offboarding:  false
         });
-        afterSpell.collaterals["DIRECT-AAVEV2-DAI"] = CollateralValues({
-            aL_enabled:   true,
-            aL_line:      100 * MILLION,
-            aL_gap:       65 * MILLION,
-            aL_ttl:       12 hours,
-            line:         0,
-            dust:         0,
-            pct:          0,
-            mat:          10000,
-            liqType:      "clip",
-            liqOn:        false,
-            chop:         1300,
-            cat_dunk:     0,
-            flip_beg:     0,
-            flip_ttl:     0,
-            flip_tau:     0,
-            flipper_mom:  0,
-            dog_hole:     0,
-            clip_buf:     10500,
-            clip_tail:    220 minutes,
-            clip_cusp:    9000,
-            clip_chip:    10,
-            clip_tip:     300,
-            clipper_mom:  0,
-            cm_tolerance: 9500,
-            calc_tau:     0,
-            calc_step:    120,
-            calc_cut:     9990,
-            offboarding:  false
-        });
```

Comment:
since this is a collateral that we will probably "reactivate" with the new Aave module (ilk naming pattern seems to match the COMPV2 one below), should we leave this here and 0 out its values?

Reply:
I would personally comment it out and leave the old values in here.

Reply:
6ec9def3c94ebf05b39c8729b7fe3b6ca0b00277